test/adapters/mongo_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+require 'flipper/adapters/mongo'
+
+class MongoTest < MiniTest::Test
+  prepend SharedAdapterTests
+
+  def setup
+    host = '127.0.0.1'
+    port = '27017'
+    collection = Mongo::Client.new(["#{host}:#{port}"], server_selection_timeout: 1, database: 'testing')['testing']
+    collection.drop
+    collection.create
+    @adapter = Flipper::Adapters::Mongo.new(collection)
+    collection.create
+  rescue Mongo::Error::OperationFailure
```

Comment:
I'd prefer to just wrap this around the drop and/or the create. The main reason, which I might be wrong on, is if mongo isn't running then I think you'll end up with @adapter not being assigned which would cause no method errors instead of mongo errors (which point you at mongo not running). Maybe just try stopping mongo and running this spec and seeing what happens? If it fails with a decent error that would make me or someone else realize mongo isn't running then I'm fine with this. If it doesn't, then perhaps moving the rescue to where it was or something better should be done. Any thoughts or did you run into anything weird here?